icons/unlicense.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Unlicense</title><path d="M12 0A12 12 0 0 0 0 12a12 12 0 0 0 12 12 12 12 0 0 0 12-12A12 12 0 0 0 12 0Zm0 2.45a9.551 9.551 0 0 1 5.557 1.79L4.24 17.557A9.551 9.551 0 0 1 2.45 12 9.551 9.551 0 0 1 12 2.45Zm0 2.816A6.735 6.735 0 0 0 5.266 12a6.735 6.735 0 0 0 .826 3.215l2.351-2.352A3.673 3.673 0 0 1 8.326 12 3.673 3.673 0 0 1 12 8.326a3.673 3.673 0 0 1 .865.115l2.348-2.347A6.735 6.735 0 0 0 12 5.266Zm7.73 1.129A9.551 9.551 0 0 1 21.55 12 9.551 9.551 0 0 1 12 21.55a9.551 9.551 0 0 1-5.605-1.82ZM17.904 8.77l-1.638 1.638h2.277a6.735 6.735 0 0 0-.639-1.638Zm-2.543 4.699A3.673 3.673 0 0 1 12 15.674a3.673 3.673 0 0 1-.879-.121L8.773 17.9a6.735 6.735 0 0 0 3.227.834 6.735 6.735 0 0 0 6.555-5.265Z"/></svg>
```

Comment:
When I derive the icon from the Wikipedia source, the SVG looks like this:

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Unlicense</title><path d="M24 12c0 6.627-5.373 12-12 12S0 18.627 0 12 5.373 0 12 0s12 5.373 12 12ZM12 2.449A9.551 9.551 0 0 0 2.449 12c0 2.09.672 4.024 1.811 5.597L17.597 4.26A9.508 9.508 0 0 0 12 2.449Zm0 19.102A9.551 9.551 0 0 0 21.551 12c0-2.09-.672-4.024-1.811-5.597L6.403 19.74A9.508 9.508 0 0 0 12 21.551Zm0-2.816a6.704 6.704 0 0 1-3.34-.885l2.32-2.32a3.674 3.674 0 0 0 4.388-2.06h3.206A6.737 6.737 0 0 1 12 18.734ZM5.265 12A6.735 6.735 0 0 1 15.34 6.15l-2.32 2.32a3.673 3.673 0 0 0-4.55 4.55l-2.32 2.32A6.704 6.704 0 0 1 5.265 12Zm13.28-1.592h-2.443L17.85 8.66c.309.54.545 1.128.695 1.748Z"/></svg>
```

It also seems to have quite a bit shorter SVG path data.

Reply:
Strange, sorry about that. Must have to do with how I combined the paths in Inkscape.